server/public/model/channel.go

```diff
@@ -418,4 +418,10 @@ type GroupMessageConversionRequestBody struct {
 	TeamID      string `json:"team_id"`
 	Name        string `json:"name"`
 	DisplayName string `json:"display_name"`
+	Header      string `json:"header"`
+}
+
+func (r *GroupMessageConversionRequestBody) PrepareForAutoConversion() {
+	r.Name = "converted-" + NewId()[:7]
```

Comment:
Why `:7`?

Reply:
I think it has very low chance of conflict and didn't want to have long team names. This should be a rather edge case.